src/Files.App/Helpers/FilePropertiesHelpers.cs

```diff
@@ -67,72 +58,73 @@ public static async Task OpenPropertiesWindowAsync(object item, IShellPage assoc
-
-				// Set content
-				propertiesWindow.Content = frame;
-				if (frame.Content is Properties properties)
-					properties.appWindow = appWindow;
-
-				// Set min size
-				propertiesWindow.MinWidth = 460;
-				propertiesWindow.MinHeight = 550;
-
-				// Set backdrop
-				propertiesWindow.Backdrop = new WinUIEx.MicaSystemBackdrop() { DarkTintOpacity = 0.8 };
 
+				var appWindow = propertiesWindow.AppWindow;
+				appWindow.Title = "PropertiesTitle".GetLocalizedResource();
 				appWindow.TitleBar.ExtendsContentIntoTitleBar = true;
-
-				// Set window buttons background to transparent
 				appWindow.TitleBar.ButtonBackgroundColor = Colors.Transparent;
 				appWindow.TitleBar.ButtonInactiveBackgroundColor = Colors.Transparent;
 
-				appWindow.Title = "PropertiesTitle".GetLocalizedResource();
 				appWindow.Resize(new SizeInt32(460, 550));
+				appWindow.SetIcon(LogoPath);
+
+				if (frame.Content is Properties properties)
+					properties.appWindow = appWindow;
+
 				appWindow.Show();
 
 				if (true) // WINUI3: move window to cursor position
 				{
 					UWPToWinAppSDKUpgradeHelpers.InteropHelpers.GetCursorPos(out var pointerPosition);
 					var displayArea = DisplayArea.GetFromPoint(new PointInt32(pointerPosition.X, pointerPosition.Y), DisplayAreaFallback.Nearest);
-					var appWindowPos = new PointInt32()
+					var appWindowPos = new PointInt32
 					{
-						X = displayArea.WorkArea.X + Math.Max(0, Math.Min(displayArea.WorkArea.Width - appWindow.Size.Width, pointerPosition.X - displayArea.WorkArea.X)),
-						Y = displayArea.WorkArea.Y + Math.Max(0, Math.Min(displayArea.WorkArea.Height - appWindow.Size.Height, pointerPosition.Y - displayArea.WorkArea.Y))
+						X = displayArea.WorkArea.X
+							+ Math.Max(0, Math.Min(displayArea.WorkArea.Width - appWindow.Size.Width, pointerPosition.X - displayArea.WorkArea.X)),
+						Y = displayArea.WorkArea.Y
+							+ Math.Max(0, Math.Min(displayArea.WorkArea.Height - appWindow.Size.Height, pointerPosition.Y - displayArea.WorkArea.Y)),
 					};
 					appWindow.Move(appWindowPos);
 				}
 			}
 			else
 			{
-				var propertiesDialog = new PropertiesDialog();
-				propertiesDialog.propertiesFrame.Tag = propertiesDialog;
-				propertiesDialog.propertiesFrame.Navigate(typeof(Properties), new PropertiesPageNavigationArguments()
+				var dialog = new PropertiesDialog();
+				dialog.propertiesFrame.Tag = dialog;
+				Navigate(dialog.propertiesFrame);
+				await dialog.ShowAsync(ContentDialogPlacement.Popup);
+			}
+
+			void Navigate(Frame frame)
+			{
+				var argument = new PropertiesPageNavigationArguments
 				{
 					Item = item,
-					AppInstanceArgument = associatedInstance
-				}, new SuppressNavigationTransitionInfo());
-				await propertiesDialog.ShowAsync(ContentDialogPlacement.Popup);
+					AppInstanceArgument = associatedInstance,
+				};
+				frame.Navigate(typeof(Properties), argument, new SuppressNavigationTransitionInfo());
 			}
 		}
+
+		private static string GetFilesLogoPath()
```

Comment:
Should this be a `constant`?

Reply:
@cinqmilleans

Reply:
I created a constant. It is also used in another place.